common/testing/virtual-threads/src/main/java/io/helidon/common/testing/virtualthreads/PinningException.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.common.testing.virtualthreads;
+
+import jdk.jfr.consumer.RecordedEvent;
+
+/**
+ * Exception used for reporting of Virtual Thread pinning detected during test.
+ */
+public class PinningException extends AssertionError {
```

Comment:
Maybe this should be `PinningAssertionError`?

Reply:
sound cool